babi/screen.py

```diff
@@ -417,7 +417,10 @@ def replace(self) -> None:
                 'replace with', history='replace', allow_empty=True,
             )
             if response is not PromptResult.CANCELLED:
-                self.file.replace(self, search_response, response)
+                try:
+                    self.file.replace(self, search_response, response)
+                except re.error:
+                    self.status.update(f'invalid replace text: {response}')
```

Comment:
so this has another problem, the user has already gone through some of the actions to say "yes I want to replace this" and only then they are prompted with the error message

ideally, they would be presented with the error message before they are subjected to menuing